src/overlays/actors/ovl_En_Viewer/z_en_viewer.c

```diff
@@ -265,87 +295,88 @@ void func_80B2A75C(EnViewer* this, GlobalContext* globalCtx) {
         }
     }
 
-    if (D_80B2CFCC != 0) {
-        D_80B2CFCC--;
+    if (sTimer != 0) {
+        sTimer--;
     }
 
-    EnViewer_GetCutsceneNextPos(this, globalCtx);
-    Actor_MoveForward(&this->actor);
+    EnViewer_UpdatePosition(this, globalCtx);
+    Actor_MoveForward(&this->actor); // has no effect, speed is 0
```

Comment:
believe this still applies gravity even if speed is 0 (also a point in favor of renaming this func from "move forward" to something else)

Reply:
Good point, however I think this actor has gravity 0? it doesn't set any and I don't think there is a default gravity besides the struct being bzero'd